blockchain/pool.go

```diff
@@ -144,12 +153,13 @@ func (pool *BlockPool) IsCaughtUp() bool {
 	receivedBlockOrTimedOut := (pool.height > 0 || time.Since(pool.startTime) > 5*time.Second)
 	ourChainIsLongestAmongPeers := maxPeerHeight == 0 || pool.height >= maxPeerHeight
 	isCaughtUp := receivedBlockOrTimedOut && ourChainIsLongestAmongPeers
-	pool.Logger.Info(Fmt("IsCaughtUp: %v", isCaughtUp), "height", pool.height, "maxPeerHeight", maxPeerHeight)
+	pool.Logger.Info(Fmt("IsCaughtUp: %v", isCaughtUp), "height", pool.height,
```

Comment:
Am assuming this is because of proper line breaks when reading on your editor ;)

Reply:
Yes, I like the 100 line limit :-) Allows me to have two emacs windows on a 13" screen.